src/Orleans.Transactions/InClusterTM/TransactionManager.cs

```diff
@@ -43,8 +132,9 @@ public class TransactionManager : ITransactionManager
         protected readonly ILogger logger;
         private bool IsRunning;
         private Task transactionLogMaintenanceTask;
-
-        public TransactionManager(TransactionLog transactionLog, IOptions<TransactionsConfiguration> configOption, ILoggerFactory loggerFactory, TimeSpan? logMaintenanceInterval = null)
+        private TransactionManagerMetrics metrics;
+        public TransactionManager(TransactionLog transactionLog, IOptions<TransactionsConfiguration> configOption, ILoggerFactory loggerFactory, ITelemetryProducer telemetryProducer,
+            Factory<NodeConfiguration> getNodeConfig, TimeSpan? logMaintenanceInterval = null)
```

Comment:
probably out of scope for this PR, but should `logMaintenanceInterval` be a part of `TransactionsConfiguration`

Reply:
yeah you are probably right, not sure why it isn't now. @jason-bragg for more opinion.

Reply:
Regarding logMaintenanceInterval, unclear.  I don't, at this time, think it should be configurable.